ui/intermittent-failures/View.jsx

```diff
@@ -73,6 +73,53 @@ const withView = (defaultState) => (WrappedComponent) => {
       }
     };
 
+    digestMessage = async (message) => {
+      const msgUint8 = new TextEncoder().encode(message);
+      const hashBuffer = await crypto.subtle.digest('SHA-256', msgUint8);
+      const hashArray = Array.from(new Uint8Array(hashBuffer));
+      const hashHex = hashArray
+        .map((b) => b.toString(16).padStart(2, '0'))
+        .join('');
+      return hashHex;
+    };
+
+    // trim off the timestamp and "TEST-UNEXPECTED-XXX | "
+    lineTrimmer = async (input) => {
```

Comment:
This function would benefit from more descriptive variable names similar to the other `lineTrimmer` function.